### PR TITLE
added EmployeeService

### PR DESCRIPTION
**Pull request by qm-sasmin**
Added EmployeeService and updated the methods in EmployeeController.

### CODE REVIEW

File: EmployeeApi/Services/EmployeeService.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        {
+            var employee = _context.Employees.Find(id);
+            return employee;
+        }
+
+        public Employee AddEmployee(Employee employee)
+        {
+            if (int.TryParse(employee.PhoneNumber, out int phoneNumber) && employee.PhoneNumber.Length == 10)
+            {
+                employee.FirstName = char.ToUpper(employee.FirstName[0]) + employee.FirstName.Substring(1);
```
Comment (qm-bhupal) [inline EmployeeApi/Services/EmployeeService.cs:29]: use ToTitleCase()

File: EmployeeApi/Services/EmployeeService.cs [hunk 2]
```diff
@@ -0,0 +1,86 @@
+﻿using EmployeeApi.Data;
+using EmployeeApi.Model;
+
+namespace EmployeeApi.Services
+{
+    public class EmployeeService
+    {
```
Comment (qm-bhupal) [inline EmployeeApi/Services/EmployeeService.cs:7]: use interface for this service

Comment (qm-sasmin) [reply EmployeeApi/Services/EmployeeService.cs:7]: Yes... Ok

File: EmployeeApi/Services/EmployeeService.cs (hunk 1 above)
Comment (qm-sasmin) [reply EmployeeApi/Services/EmployeeService.cs:29]: Yes.. I will change it.

**Pull request merged**